src/content/1.7/modules/introduction.md

```diff
@@ -21,6 +21,10 @@ the cart or the product sheet, when displaying the current stock, etc.).
 Specifically, a hook is a shortcut to the various methods available from
 the `Module` object, as assigned to that hook.
 
+For obvious security reasons, during validation, we do not accept any call
```

Comment:
What is `obvious` for you can be difficult to understand to someone else, and should be removed.

Reply:
done